internal/admin/dashboard/templates/audit-pane.html

```
@@ -0,0 +1,25 @@
+{{define "audit-pane"}}
+<section class="audit-pane">
+    <div class="audit-pane-head">
+        <h4 x-text="{{.}}.title"></h4>
+        <button class="pagination-btn" @click.prevent="copyAuditJSON({{.}}.copyBody, $event)">Copy Body</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)